src/lib/community.js

```diff
@@ -0,0 +1,3 @@
+console.log('We have an active developer community on Discord. '
+            + 'You can find it on https://discord.gg/crawlee-apify-801163717915574323 or when you write apify -h.',
```

Comment:
Because of what you have in the PR description it should be probably:

```suggestion
            + 'You can find it on https://discord.gg/crawlee-apify-801163717915574323.',
```